### PR TITLE
Stg

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: announcements.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
...
 | --- | --- | 
+| 19 March 2024 | SEED team will be implementing the new VPC Migration for the Production environment on 23 March, Saturday, with a downtime from 7:00 to 18:00 SGT.<br><br>The following changes will be implemented:<br> - Intermittent access to SGTS.<br> - Intermittent access or outdated content in the SEED dashboard.<br> - Onboarding processes will be unavailable (offboarding remains unaffected).<br> - The Tanium console will be inaccessible for support needs.<br><br>**Impact**:<br/>There should be minimal impact to users during the migration window. However, please plan for potential intermittent access issues to SGTS and dashboard services. Onboarding processes will be paused during this time, but offboarding will not be impacted. Support through the Tanium console will be temporarily unavailable.<br>If you are still encountering issues, please refer to the [troubleshooting guide](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/support/troubleshooting-issues). <br><br>**For more assistance**:<br/> Create an [incident support request](https://go.gov.sg/seed-techpass-support).|
```
Comment (cricketer94) [inline announcements.md:5]:
```suggestion
| 19 March 2024 | SEED team will be conducting migration and maintenance on their backend on 23 March, Saturday, with a downtime from 7:00 to 18:00 SGT.<br><br>The following will services will be affected during this period:<br> - Intermittent access to SGTS.<br> - Intermittent access or outdated content in the SEED dashboard.<br> - Onboarding processes will be unavailable (offboarding remains unaffected).<br> - The Tanium console will be inaccessible for support needs.<br><br>**Impact**:<br/>There should be minimal impact to users during the migration window. However, please plan for potential intermittent access issues to SGTS and dashboard services. Onboarding processes will be paused during this time, but offboarding will not be impacted.<br>If you are still encountering issues, please refer to the [troubleshooting guide](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/support/troubleshooting-issues). <br><br>**For more assistance**:<br/> Create an [incident support request](https://go.gov.sg/seed-techpass-support).|
```

Comment (cricketer94) [inline announcements.md:5]: @eu-tee not sure if this comment on Tanium console not available for support needs to be in the announcement for users. I can just inform L2 and L3 tomorrow?

**Review by cricketer94 — COMMENTED** [review]
@kpslp just some minor changes to remove some of the unncessary technical details

File: announcements.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 | <div style="width:150px"><b>Date</b></div>  | Announcement |
 | --- | --- | 
+| 19 March 2024 | SEED team will be implementing the new VPC migration for the production environment on 23 March, Saturday, with a downtime from 7:00 to 18:00 SGT.<br><br>The following changes will be implemented:<br> - Intermittent access to SGTS.<br> - Intermittent access or outdated content in the SEED dashboard.<br> - Onboarding processes will be unavailable (offboarding remains unaffected).<br><br>**Impact**:<br/>There should be minimal impact to users during the migration window. However, please plan for potential intermittent access issues to SGTS and dashboard services. Onboarding processes will be paused during this time, but offboarding will not be impacted. Support through the Tanium console will be temporarily unavailable.<br>If you are still encountering issues, please refer to the [troubleshooting guide](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/support/troubleshooting-issues). <br><br>**For more assistance**:<br/> Create an [incident support request](https://go.gov.sg/seed-techpass-support).|
```
[the remark this answers is not in the document]
Comment (kpslp) [reply announcements.md]: done

**Review by eu-tee — APPROVED** [review]

**Pull request merged**